File: src/boot.c
#include <stdio.h>
#include <string.h>
#include "calls.h"
#include <Windows.h>
void bootk() {
    char bootinput[100];

    printf("==CCOS== select a option to boot: ccos\n");
    scanf("%99s", bootinput);

    if (strcmp(bootinput, "ccos") == 0) {
        Sleep(2000);
        printf("booting to %s... please wait..\n", bootinput);
        printf("booted! initializing...\n");
        kernel();

    } else {
          printf("no command such %s\n", bootinput);

    }
    

}

File: CCOS/boot.py
import os
import time
from kernel import mainsys


def bootloader():
    print("==CCBoot active please select a system==")
    print("CCOS")
    bootinput = input()
    if bootinput == "ccos":
        print("booting up....")
        time.sleep(3)
        print("booted!")
        time.sleep(1)
        os.system('cls')
        mainsys()
    if bootinput == "bootdebug":
        mainsys()

    else:
        print("no command!")
        bootloader()


if __name__ == "__main__":
    bootloader()
